guides/common/modules/con_package-mode-and-image-mode-hosts.adoc

```diff
@@ -7,8 +7,11 @@ It classifies all other hosts as package mode hosts.
 Image mode simplifies enterprise operations by enabling development, operations, and solution providers to use the same container-native tools and techniques to manage both applications and the underlying operating system.
 
 In image mode, users can install and manage packages, but with limitations. 
+Job templates, depending on the implementation and category, will either enable the `bootc usr-overlay` or pass the `--transient` flag to ensure that packages are installed correctly.
```

Comment:
It would be nice to rephrase this as an action to be performed by users. Written like this, I'm not entirely sure what myself, as a potential user, should do. Just guessing, but:

```suggestion
When creating job templates, enable the `bootc usr-overlay` command or pass the `--transient` option to ensure that packages are installed correctly.
```
One note: I'm suggesting `the [...] command` because that seemed to be missing and s/flag/option because by convention, we don't use "flag" here.

Reply:
This line was meant to be more informational noting that our job templates will either enable the usr-overlay or pass the transient flag into dnf depending on if the host is an image mode host or not.

Reply:
So, there's no action that the user has to take besides understanding that their packages will be installed transiently (which, well, we do have warnings in the UI about it, but it felt nice putting it here too).

Reply:
Okay, can you please think of a way to avoid "will enable" (to avoid future tense here) and add the missing qualifying word ("the `bootc usr-overlay` something")? That's just to comply with our style guide.

Reply:
I've updated it now.